qemu/tests/qemu_guest_agent.py

```diff
@@ -2140,6 +2142,95 @@ def result_check(flag, write_timeout, session):
                               "stamp in guest file." % write_timeout)
         result_check("thaw", write_timeout, session)
 
+    @error_context.context_aware
+    def gagent_check_fstrim(self, test, params, env):
+        """
+        Execute "guest-fstrim" command via guest agent.
+
+        Steps:
+        1) boot up guest with scsi backend device
+        2) init and format the data disk
+        3) get the original blocks of data disk
+        4) create fragment in data disk
+        5) check the used blocks
+        6) execute fstrim cmd via guest agent
+        7) check if the used blocks is decreased
```

Comment:
I am not sure the necessity of checking different disk format, as polarion test both qcow2 and raw.